### PR TITLE
Added DevOps to Roles in Tech

### PR DESCRIPTION
**Pull request by pydevsg**
A short description to start with the role of becoming a  DevOps in today's world.
Fixed Issue #93.

### CODE REVIEW

File: roles-in-tech/dev-ops.md [hunk 1]
```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
...
-- Example
+- **DevOps is a set of automated tools.**
+- **DevOps procedures change from project to project.**: 
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+Devops consists of a lot of tools. From learning *Git VCS* to deploying code in *Docker*, one need to learn everything to become a good DevOps Engineer.
```
Comment (manaswinidas) [inline roles-in-tech/dev-ops.md]: It's DevOps

**Comment by manaswinidas** [comment]
Is it a duplicate PR, @pydevsg ?

File: roles-in-tech/dev-ops.md [hunk 2]
```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+*One of the motivation to join DevOps is its `Fast Career Growth`.* 
+You can become a **Release Manager**, **Project Manager**, **Automation Architect** or even a **DevOps Evangelist** by learning DevOps.
+
+<h2>Any One Can Learn DevOps</h2>
+>People from different backgrounds can learn DevOps. Even a fresher with basic knowledge of Linux and one scripting language can learn DevOps.
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: Please give an example of a company that uses DevOps well or depends on it a lot, and how they do it.

File: roles-in-tech/dev-ops.md (hunk 1 above)
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: “One need to learn everything”? Can you please phrase that differently, like one needs to learn a wide variety of tools?

File: roles-in-tech/dev-ops.md [hunk 3]
```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
...
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+Devops consists of a lot of tools. From learning *Git VCS* to deploying code in *Docker*, one need to learn everything to become a good DevOps Engineer.
+[Edureka](https://www.edureka.co/devops)
 
 ### Independent Practice
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+[DevOps best practices](https://devops.com/design-devops-best-practices/)
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: What should students do with this link? Read all of it thoroughly? Spend 5 minutes clicking around? Do an exercise?

File: roles-in-tech/dev-ops.md [hunk 4]
```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
...
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+[AWS Certified DevOps Engineer - Professional](https://aws.amazon.com/certification/certified-devops-engineer-professional/)
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: Same as my last comment for independent practice

File: roles-in-tech/dev-ops.md [hunk 5]
```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
...
 
 
 ### Check for Understanding
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+For Better Understanding of *DevOps* , one should do continuous unit testing after deploying it on a suitable platform.
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: Have you told them what continuous integration or testing is yet? They won’t learn how to test here, but maybe you can explain or show them a few CI tools and what they do.

File: roles-in-tech/dev-ops.md (hunk 1 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: Okay, @manaswinidas .

File: roles-in-tech/dev-ops.md (hunk 2 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: Okay, I am adding this on the slide as well as in my content.

File: roles-in-tech/dev-ops.md (hunk 1 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: Yes, I am changing it.

File: roles-in-tech/dev-ops.md (hunk 3 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: Spend 5 minutes clicking would be preferred.

File: roles-in-tech/dev-ops.md (hunk 4 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: To be a certified DevOps Engineer , one should refer to the above.

File: roles-in-tech/dev-ops.md (hunk 5 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: Yes, I have added CI/CD part in 

> Materials
section and also in the ppt.

**Comment by pydevsg** [comment]
Thanks @alodahl.
Please review this PR @manaswinidas.

File: roles-in-tech/dev-ops.md [hunk 6]
```diff
@@ -17,75 +13,84 @@ Here are links to lessons that should be completed before this lesson.
...
 ### Independent Practice
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+Spend 5 minutes on [DevOps best practices](https://devops.com/design-devops-best-practices/)
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
-
+To become a certified DevOps Engineer one should refer to [AWS Certified DevOps Engineer - Professional](https://aws.amazon.com/certification/certified-devops-engineer-professional/)
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: Move this content to supplemental materials and remove challenge header, please.

File: roles-in-tech/dev-ops.md [hunk 7]
```diff
@@ -17,75 +13,84 @@ Here are links to lessons that should be completed before this lesson.
...
 
 ### Check for Understanding
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+For Better Understanding of *DevOps* , one should do continuous unit testing after deploying it on a suitable platform.
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: Move this advice to lesson and remove the check for understanding header, please.

Comment (alodahl) [reply roles-in-tech/dev-ops.md]: Besides addressing our review comments, please be sure to see my comment in your slideshow, and follow the stying guidelines found in #600 . Also, we have several outlines on testing - search the repo for anything with “testing” in the file name, and consider adding them to ‘prerequisites’.

File: roles-in-tech/dev-ops.md [hunk 8]
```diff
@@ -17,75 +13,84 @@ Here are links to lessons that should be completed before this lesson.
...
-- Example
+- **DevOps is a set of automated tools.**
+- **DevOps procedures change from project to project.**: 
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+DevOps consists of a lot of tools. From learning *Git VCS* to deploying code in *Docker*, one need to learn a variety of skills to become a good DevOps Engineer.
```
[the remark this answers is not in the document]
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: I am removing the Guided Practice part as it has no practical implementation.

File: roles-in-tech/dev-ops.md (hunk 6 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: I have done the same.

File: roles-in-tech/dev-ops.md (hunk 7 above)
Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: Those things are cleared in the ppt file @alodahl and DevOps doesn't requires such prerequisites.

**Comment by pydevsg** [comment]
@alodahl and @manaswinidas please do review my PR.

**Comment by manaswinidas** [comment]
I think it will be awesome if you could include some hands-on courses on the tools mentioned in the PR and/or assessments. @alodahl what do you think?

File: roles-in-tech/dev-ops.md [hunk 9]
```diff
@@ -17,75 +13,75 @@ Here are links to lessons that should be completed before this lesson.
...
+While many industry leaders have successfully deployed DevOps to achieve faster speed-to-market with higher quality software and a first-class customer experience, there continues to be some fundamental misconceptions about what DevOps really means.
 
+- **DevOps is a set of automated tools.**
+- **DevOps procedures change from project to project.**: 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+[DevOps lessons from Edureka](https://www.edureka.co/devops)
```
Comment (alodahl) [inline roles-in-tech/dev-ops.md]: Which lesson(s) should they do if they have 5-15 minutes to try it? Describe why this is a good resource.

Comment (pydevsg) [reply roles-in-tech/dev-ops.md]: I have changed my PR according to your requirements @alodahl and @manaswinidas .

**Pull request merged**